proxy/http/HttpTransact.cc

```diff
@@ -3138,8 +3146,12 @@ HttpTransact::HandleCacheOpenReadMiss(State *s)
       }
     }
     build_request(s, &s->hdr_info.client_request, &s->hdr_info.server_request, s->current.server->http_version);
-
-    s->next_action = how_to_open_connection(s);
+    // TODO doesn't make sense to set this here, find the right place
```

Comment:
Is this so? If so, where do we put it? If we're keeping it here, then the comment probably should go ?